projects/kit/components/arrow/arrow.providers.ts

```diff
@@ -0,0 +1,29 @@
+import {Provider} from '@angular/core';
```

Comment:
What do you think?
`arrow.providers.ts` -> `arrow-mode.options.ts`

Reply:
Let's keep it as is, it is not really an option, it's a regular DI token for particular behavior. I just think it makes sense to keep it here, rather than in `tokens` to simplify secondary entry points.